### DISCUSSION TITLE
X-T5 Owner - It works! Confirmation. And GPS location possibilities?

### DISCUSSION THREAD

**Discussion by stulevine** [gkoh/furble#42] [General]
Hi @gkoh. Just wanted to let the you all know that I just installed furble to a new M5stack M5Stick C Plus ESP32-PICO-D4 Mini IoT Development Kit device purchased on [Amazon.com](https://www.amazon.com/dp/B082Y6YTY5?psc=1&ref=ppx_yo2ov_dt_b_product_details) and it works just perfectly with my new Fujifilm X-T5 with firmware v1.0.2 (just released this morning).  Nice alternative to using my iPhone.  The only other thing that would be great is if the M5Stick supported GPS locations sourcing.  The X-T5 can use the CamRemote app on the iPhone when connected view BT to derive the GPS location and add it to the photo meta data.  I'd be curious if this could be accomplished using the WiFi module of the M5Stick?  Curious if anyone has any thoughts on this.  I guess, if the M5Stick does not have GPS support, that I could purchase a GPS module and connect it to the GPIO pins and see how difficult it would be to support location transmission to the camera.

Stuart

Update: M5Stack does make a GPS unit that plugs into the M5Stick Grove connector interface and works with M5Core.  Here is the [product link](https://shop.m5stack.com/products/mini-gps-bds-unit?variant=16804810784858) I might order it and see if it's possible to send GPS coordinates to the camera.  I think the camera will poll the device for that info if enabled in the setup menu on the camera.  Any insights would be greatly appreciated.

**Comment by gkoh** [comment]
Hello @stulevine, thanks for the report!
That's 2 confirmed X-T5 successes now, and this one with the very latest firmware.

Hmm, GPS is a good idea, the answer is, I don't know.
From my analysis of the Bluetooth protocol, there is no support for passing geotag data.
I strongly suspect we would need to swap to a WiFi connection and use that, there's been some work [here](https://github.com/hkr/fuji-cam-wifi-tool).

I will investigate further when I get some time (in the mean time I might also order that M5Stack GPS unit).

**Comment by gkoh** [comment]
I had a poke around, seems like I'm wrong, this is good :slightly_smiling_face: 
According to the various sources, including this [random](https://www.fujirumors.com/tested-fujifilm-real-time-geotagging-via-bluetooth-quick-guide-tips-auto-image-transfer/) one, location sync is supported via Bluetooth only.

I have ordered the GPS unit from M5Stack and will experiment in the meantime.

**Reply by stulevine** [reply]
@gkoh that's good news.  I thought it was via BT because the CamRemote app never makes a WiFi connection to the camera when syncing the location data (geotagging the RAW files).  It only seems to require WiFi when transmitting files from the camera to the phone.  I also ordered my mini GPS from M5Stack yesterday.  Unfortunately, since I'm in the US, it will take anywhere from 5-20 days to get here via Air Post.  Oh well.  I guess we will have plenty of time to figure this out.  Thanks again for looking into this.  It will be nice to use the M5StickC as my geo locator with the X-T5.  Then I don't have to have my phone with me on my photo safaris.  I use an Apple Watch for my walks and leave the phone at home.

**Reply by stulevine** [reply]
For anyone else who wants to be able to carry the MStick5C with a lanyard or on an s-hook, I found this [3D Printable case](https://www.thingiverse.com/thing:3709069/comments) for the device.  I was able to request it be printed at my local library.  Cost me all of $1.28 and I didn't have to wait for shipping.

**Comment by gkoh** [comment]
This is now #43.
